### ISSUE TITLE
Upgrade to latest code-server (4.4.0)

### ISSUE THREAD

**Issue by marktolson** [martinussuherman/alpine-code-server#1]
Is there any chance you can upgrade the code-server version to 4.4.0. I tried locally but the terminal breaks due to an inability to load the PTY node module.

**Comment by leyume** [comment]
@marktolson hello. Were you able to resolve this problem?

**Comment by marktolson** [comment]
@leyume yep use my alpine-code-server repo if you want it to work with 4.4.0.

**Comment by marktolson** [comment]
https://github.com/marktolson/alpine-code-server

**Comment by leyume** [comment]
@marktolson Niceee. Thank you indeed. I will check it out

**Comment by martinussuherman** [comment]
@marktolson sorry been away for a long time, glad you could get it to work. I'll try catch up with newer code-server version when I got the time. Cheers ...
